src/components/EventDetails.tsx

```diff
@@ -0,0 +1,84 @@
+import { ReactElement } from "react";
+import Frame from ".././assets/Frame.svg";
+import LinkSVG from ".././assets/Link.svg";
+import MeatballMenuSVG from ".././assets/MeatballMenu.svg";
+import PinSVG from ".././assets/Pin.svg";
+import TextIconSVG from ".././assets/TextIcon.svg";
+import { Event } from "./EventCard";
+
+export type EventDetailsProps = {
+  event: Event;
+};
+
+const EventDetails = ( {event}:EventDetailsProps): ReactElement => {
```

Comment:
Lol I'm so sorry for leaving so many nitpicks lol 😭 But I would space this out like so for code readability

```suggestion
const EventDetails = ( { event } : EventDetailsProps): ReactElement => {
```